Makefile

```diff
@@ -191,7 +191,16 @@ $(dockerfiles): $(dockertemplates)/*.Dockerfile.in $(dockerlib)/*.docker
 clean-dockerfiles:
 	@rm -f build/docker/*.Dockerfile
 
-dockerfiles: $(dockerfiles)
+dockerfiles: clean-dockerfiles $(dockerfiles)
+
+check-dockerfiles: dockerfiles
+	@GITDIFF=`git diff build/docker` ; \
+	if [ -n "$$GITDIFF" ]; then \
+		echo "Changes detected is Dockerfiles:" && \
```

Comment:
typo: is -> in

Can we mention that Dockerfiles are out if sync with .in files somehow?

Reply:
Attempt made in second patch (old work squashed to 1st)